pkg/api/types_test.go

```diff
@@ -372,6 +373,7 @@ func TestAdminAPIParity(t *testing.T) {
 
 	// TODO: why don't we just include all of these in the admin type?
 	notInAdmin := []*regexp.Regexp{
+		regexp.MustCompile(`^\.Config\.ClusterVersion$`),
```

Comment:
this should be in the admin api, right? (r/o)

Reply:
I think no. I would not want to be able to change it as part of AdminAPI. AdminAPI would initiate the upgrade process, and as part of the new code in RP it will generate new version field. This way version always corresponds only to code we vendor/release. 

Do you see a need to get this changed?

Reply:
Ou, read only...